### PR TITLE
Add README and update UI img for haiku_generator

### PR DESCRIPTION
**Pull request by windmaple**
Add README and update UI img for haiku_generator

### CODE REVIEW

File: haiku_generator/finished/README.md [hunk 1]
```diff
@@ -1,19 +1,3 @@
...
-
-A few resources to get you started if this is your first Flutter project:
-
-- [Lab: Write your first Flutter app](https://docs.flutter.dev/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://docs.flutter.dev/cookbook)
-
-For help getting started with Flutter development, view the
-[online documentation](https://docs.flutter.dev/), which offers tutorials,
-samples, guidance on mobile development, and a full API reference.
+This folder contains the code for [Generative AI](https://developers.generativeai.google/) + Flutter codelab.
```
Comment (domesticmouse) [inline haiku_generator/finished/README.md:3]: Nit: please add empty line at end of file

File: haiku_generator/step0/README.md [hunk 2]
```diff
@@ -1,19 +1,3 @@
...
-
-A few resources to get you started if this is your first Flutter project:
-
-- [Lab: Write your first Flutter app](https://docs.flutter.dev/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://docs.flutter.dev/cookbook)
-
-For help getting started with Flutter development, view the
-[online documentation](https://docs.flutter.dev/), which offers tutorials,
-samples, guidance on mobile development, and a full API reference.
+This folder contains the code for [Generative AI](https://developers.generativeai.google/) + Flutter codelab.
```
Comment (domesticmouse) [inline haiku_generator/step0/README.md:3]: Ditto

File: haiku_generator/step1/README.md [hunk 3]
```diff
@@ -1,19 +1,3 @@
...
-
-A few resources to get you started if this is your first Flutter project:
-
-- [Lab: Write your first Flutter app](https://docs.flutter.dev/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://docs.flutter.dev/cookbook)
-
-For help getting started with Flutter development, view the
-[online documentation](https://docs.flutter.dev/), which offers tutorials,
-samples, guidance on mobile development, and a full API reference.
+This folder contains the code for [Generative AI](https://developers.generativeai.google/) + Flutter codelab.
```
Comment (domesticmouse) [inline haiku_generator/step1/README.md:3]: Ditto

**Review by domesticmouse — APPROVED** [review]

File: haiku_generator/step1/README.md (hunk 3 above)
Comment (windmaple) [reply haiku_generator/step1/README.md]: done

**Review by domesticmouse — APPROVED** [review]

**Comment by domesticmouse** [comment]
Feel free to land on green CI. Or bug me to land it if you can't.

**Comment by windmaple** [comment]
I'm not authorized to merge

**Comment by domesticmouse** [comment]
There's nothing here that can turn CI red, landing.

**Pull request merged**